Xamarin.Essentials/Screenshot/Screenshot.uwp.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.IO;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Threading.Tasks;
+using Windows.Graphics.Imaging;
+using Windows.Storage;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Screenshot
+    {
+        internal static bool PlatformCanCapture => true;
+
+        static async Task<ScreenshotFile> PlatformCaptureAsync()
+        {
+            var element = Window.Current.Content as FrameworkElement;
+            var fileOnDisk = await ApplicationData.Current.LocalCacheFolder.CreateFileAsync(Path.ChangeExtension(Path.GetRandomFileName(), ".png"));
```

Comment:
To be consistent, we should probably use `Path.Combine(FileSystem.CacheDirectory, Path.GetTempFileName());` like Android.

Reply:
I actually prefer the uwp implementation over all others, as this allows the user to remove the cached data by cleaning app data.

Reply:
I like this more as the user can clear application cache directly